### PR TITLE
Update edgeswitch.yaml to add new UBNT OID

### PR DESCRIPTION
**Pull request by kanokc**
change type: from network to network_switch per
https://community.librenms.org/t/help-with-definitions-yaml-files/16239

add new OID for UBNT device per
https://community.librenms.org/t/edgeswitch-os-discovery-fix/18831

Please give a short description what your pull request is for

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [ ] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/edgeswitch.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 os: edgeswitch
 text: EdgeSwitch
-type: network
```
Comment (PipoCanaja) [inline includes/definitions/edgeswitch.yaml:3]: network_switch does not exist. Please revert to network as it was before.

File: includes/definitions/edgeswitch.yaml [hunk 2]
```diff
@@ -11,21 +11,21 @@ over:
...
             - .1.3.6.1.4.1
+            - .1.3.6.1.4.1.8072.3.2.10
+            - .1.3.6.1.4.1.27282.3.2.10            
         sysDescr_regex:
             - '/^EdgeSwitch/'
             - '/^EdgePoint/'
             - '/^USW[ -]/'
             - '/^UBNT US/'
             - '/^US[ -]/'
-    -
```
Comment (PipoCanaja) [inline includes/definitions/edgeswitch.yaml:25]: This change is not correct either. The move of the sysObjectIDs that were below up to the top changes the behaviour. 
Adding `- .1.3.6.1.2.1.1.1.0` is the only thing you need to do in this PR.

**Review by PipoCanaja — CHANGES_REQUESTED** [review]

**Comment by PipoCanaja** [comment]
Hi @kanokc 
I did rollback to the minimum change to add the new OID. Type should remain "network". 
Please add test data as described above.
Thx

**Comment by kanokc** [comment]
Test data attached

[edgeswitch_edgesw.txt](https://github.com/user-attachments/files/16040172/edgeswitch_edgesw.txt)

**Comment by murrant** [comment]
Wait... Test data shows this is wrong and also already detected????

**Pull request merged**